### PR TITLE
Fixed Ampersand issues in selecting products from category having &

### PR DESCRIPTION
**Pull request by santosh-subrahmanya**

### CODE REVIEW

**Comment by santosh-subrahmanya** [comment]
Please approve the changes done to fix the ampersand issue in selecting products from categories

File: Shopkick/src/app/admin/admin.component.css [hunk 1]
```diff
@@ -6,4 +6,8 @@
 
 .card[_ngcontent-c5] {
     width: 700px;
+}
+
+.card[_ngcontent-c11] {
+    width: 700px;
```
Comment (mdodballapur) [inline Shopkick/src/app/admin/admin.component.css:12]: Shouldn't 800px

File: src/main/java/com/products/barcodes/scanners/services/impl/BarcodesServiceImpl.java [hunk 2]
```diff
@@ -28,7 +28,7 @@
 		Query query = new Query();
 		query.addCriteria(Criteria.where(ApplicationConstants.COMPANY).is(companyName));
 		if(null != category && !category.equals(ApplicationConstants.ALL)) {
-			query.addCriteria(Criteria.where(ApplicationConstants.CATEGORY).is(category));
+			query.addCriteria(Criteria.where(ApplicationConstants.CATEGORY).regex(category));
```
Comment (mdodballapur) [inline src/main/java/com/products/barcodes/scanners/services/impl/BarcodesServiceImpl.java:31]: Will regex work ?

**Review by mdodballapur — APPROVED** [review]
LGTM (Looks Good To Me)

File: src/main/java/com/products/barcodes/scanners/services/impl/BarcodesServiceImpl.java (hunk 2 above)
Comment (santosh-subrahmanya) [reply src/main/java/com/products/barcodes/scanners/services/impl/BarcodesServiceImpl.java:31]: Yes it works

File: Shopkick/src/app/admin/admin.component.css (hunk 1 above)
Comment (santosh-subrahmanya) [reply Shopkick/src/app/admin/admin.component.css:12]: Will see this change required

**Pull request merged**